aci/data_source_aci_vnsredirectdest.go

```diff
@@ -44,19 +45,16 @@ func dataSourceAciDestinationofredirectedtraffic() *schema.Resource {
 				Optional: true,
 				Computed: true,
 			},
-
-			"name_alias": &schema.Schema{
+			"pod_id": &schema.Schema{
 				Type:     schema.TypeString,
 				Optional: true,
 				Computed: true,
 			},
-
-			"pod_id": &schema.Schema{
+			"relation_vns_rs_redirect_health_group": &schema.Schema{
 				Type:     schema.TypeString,
 				Optional: true,
```

Comment:
shouldn't this be computed?

Reply:
It didn’t make difference in the state file.